cache/fast_lru_cache.h

```diff
@@ -84,13 +115,8 @@ struct LRUHandle {
     if (metadata_charge_policy != kFullChargeCacheMetadata) {
       return 0;
     } else {
-#ifdef ROCKSDB_MALLOC_USABLE_SIZE
-      return malloc_usable_size(
-          const_cast<void*>(static_cast<const void*>(this)));
-#else
-      // This is the size that is used when a new handle is created.
-      return sizeof(LRUHandle) - 1 + key_length;
-#endif
+      return sizeof(LRUHandle);  // TODO Is this the right metadata accounting
```

Comment:
The "#ifdef ROCKSDB_MALLOC_USABLE_SIZE" should be kept. Is there a reason to remove it?

Reply:
Because the size of the structure is not dynamic any more (keys are always 16 bytes long).

Reply:
This section is about whether to use "malloc_usable_size".

Reply:
There is no dynamic metadata charge per entry added, so `CalcMetaCharge` doesn't belong. If metadata is charged, we should do that for the whole table at construction time. There are complicated arguments for and against using `malloc_usable_size` on large objects whose internal fragmentation will likely be dominated by unused pages; at present *code consistency* favors using `malloc_usable_size` anyway on large objects.

Reply:
> anyway on large objects

For example, without `optimize_filters_for_memory`, "full filters" can be MB in size but quite randomly sized.